tests/test_analysis.py

```diff
@@ -64,6 +64,25 @@ def test_can_find_python_files_in_dot(self):
             actual_suffix = os.path.splitext(python_path)[1]
             assert actual_suffix == ".py"
 
+    def test_can_find_files_from_cloned_git_link(self):
+        git_link = "https://github.com/roskakori/pygount.git"
+        with analysis.SourceScanner([git_link]) as scanner:
+            actual_paths = list(scanner.source_paths())
+            assert actual_paths != []
+
+    def test_can_split_repo_link_and_branch(self):
```

Comment:
If this tests the function `repo_arguments`, the test name should say so. Suggestion: `test_can_split_repo_arguments`.

Or you could rename the function, which would probably be a good idea because `arguments` is very generic. `repo_link_and_branch`.

Although in the git docs it's always called "remote URL", so `repo_url_and_tag` seems even better.

Reply:
Changed in [86cbce5](https://github.com/roskakori/pygount/pull/111/commits/86cbce5e2f49caf80242b97c3aa5c0708b4abf03) the function to `valid_repo_url_and_tag` by merging `is_valid_git_link` and `repo_arguments`.

Also renamed all `git_link` to `git_url`

Reply:
Changed to GitStorage